apps/testing-javascript/src/data/mux-assets.ts

```diff
@@ -0,0 +1,159 @@
+// Running this script:
+//
+// Ensure you have `dataFilePath` set to the location of the formatted export
+// of Testing JavaScript data. Also, be sure to set `MUX_ACCESS_TOKEN` and
+// `MUX_SECRET_KEY` in your `.env.local` environment variables file.
+//
+// Execute the Mux Asset Upload script with:
+//
+// ```
+// npx ts-node --skipProject src/data/mux-assets.ts
+// ```
```

Comment:
this is the script that fed all of the S3 mezzanine video files into Mux and then captures the corresponding playback IDs. this gets used with the Sanity Import script when creating all of the `VideoResource` entities.